flask_admin/contrib/sqla/view.py

```diff
@@ -555,6 +555,33 @@ def init_search(self):
 
         return bool(self.column_searchable_list)
 
+    def search_placeholder(self):
+        """
+            Return search placeholder.
+
+            For example, if set column_labels and column_searchable_list:
+
+            class MyModelView(BaseModelView):
+                column_labels = dict(name='Name', last_name='Last Name')
+                column_searchable_list = ('name', 'last_name')
+
+            placeholder is: "Search: Name, Last Name"
+        """
+        if not self.column_searchable_list:
+            return 'Search'
+
+        placeholders = []
+
+        for searchable in self.column_searchable_list:
+            if isinstance(searchable, InstrumentedAttribute):
+                placeholders.append(
+                    self.column_labels.get(searchable.key, searchable.key))
+            else:
+                placeholders.append(
+                    self.column_labels.get(searchable, searchable))
+
+        return 'Search: %s' % u', '.join(placeholders)
```

Comment:
@illume Do you have any insight here on what will happen if we use lazy_gettext in column_labels, and then use self.column_labels.get, will it get the already translated label?

gettext might not necessary here.